app/views/chefs/ingredients/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Chef <%= "#{@chef.name}'s" %> Ingredients</h1>
+<% @chef.chef_ingredients.each do |ingredient| %>
+<p><%= ingredient %></p>
+<% end %>
```

Comment:
I made some comments about an improved implementation in your chef ingredients controller.